tasks/dials.ts

```diff
@@ -115,9 +115,9 @@ const dialsDetailsToString = (dialsDetails: Array<DialDetails>) =>
     dialsDetails
         .map(
             (dd, i) =>
-                `${dialNames[i].padStart(21)}\t${usdFormatter(dd.voteWeight, 18, 5, 2)}\t${usdFormatter(dd.distributed)}\t${usdFormatter(
-                    dd.donated,
-                )}\t ${usdFormatter(dd.rewards)}`,
```

Comment:
you can remove the index "i" , it is not used.
but question how it avoids the disabled ones ? i would expect a filter or similar

Reply:
right, forgot. The `i` was used before.

The filter happens in the calculation, disabled dials are skipped in the loop. Line 225